crates/verify/src/etherscan/mod.rs

```diff
@@ -110,16 +109,15 @@ impl VerificationProvider for EtherscanVerificationProvider {
                         || resp.result.starts_with("The address is not a smart contract")
                     {
                         warn!("{}", resp.result);
-                        return Err(eyre!("Could not detect the deployment."));
+                        return Err(eyre!("Could not detect deployment: {}", resp.result));
                     }
 
-                    warn!("Failed verify submission: {:?}", resp);
                     sh_err!(
-                        "Encountered an error verifying this contract:\nResponse: `{}`\nDetails:
-                        `{}`",
+                        "Failed to verify contract - Response: `{}`, Details: `{}`",
```

Comment:
```suggestion
                        "Encountered an error verifying this contract:\nResponse: `{}`\nDetails:
                        `{}`",
```

Restoring this seems preferable, splitting this out into multiple lines improves readability